lib/std/crypto/aes_cmac.zig

```diff
@@ -0,0 +1,170 @@
+/// RFC 4493 - The AES-CMAC Algorithm
+/// https://www.rfc-editor.org/rfc/rfc4493
+const std = @import("std");
+const Aes = std.crypto.core.aes;
+const Aes128 = Aes.Aes128;
+
+pub fn AesCmac() type {
+    return struct {
+        const Self = @This();
+        key: [Aes.Block.block_length]u8,
+
+        /// Create a new cmac context with the given key.
+        pub fn init(key: [Aes.Block.block_length]u8) Self {
+            return Self{ .key = key };
+        }
+
+        /// Generates a cmac from the given message.
+        pub fn generate(self: Self, msg: []const u8) [Aes.Block.block_length]u8 {
+            var ctx = Aes128.initEnc(self.key);
```

Comment:
Key expansion is slow, but can be reused. So that should be moved to `init()`, not the key.

Reply:
done